www/searchutil.php

```diff
@@ -1039,6 +1049,11 @@ function doSearch($db, $term, $searchType, $sortby, $limit, $browse)
               $baseOrderBy
             $limit";
 
+    $logging_level = 0;
```

Comment:
Is this meant to be committed?

Reply:
Yes… I kept having to manually re-add it. I'll use it a bunch in future PRs.